src/test/java/io/github/thebusybiscuit/slimefun4/core/services/TestBlockDataService.java

```diff
@@ -1,54 +0,0 @@
-package io.github.thebusybiscuit.slimefun4.core.services;
```

Comment:
There's no global key anymore.... so this test is kinda impossible. Maybe there's a better way - you're a smart reviewer, you tell me how to do this cause I'm an idiot  :))